Feature/50 add validations for maven release (#80)
exasol/release-robot: doc/user_guide/maven_central_release_template.md

# Template of `maven_central_release.yml`

> Copy the content into a file `/.github/workflows/maven_central_release.yml` in your project.    

Reviewer: ```suggestion
Create a `/.github/workflows/maven_central_release.yml` file in your project with the following content:
```
